fe/src/components/campaign/fund-form.vue

```diff
@@ -24,18 +24,33 @@
           Supporting a project for no reward, just because it speaks to you.
         </p>
       </div>
-      <button type="submit" class="btn-gradient-hr">Fund Campaign</button>
+      <BaseButton
+        type="submit"
+        loading-class="bg-disabled h-9 px-4 rounded-[6px] text-white"
+        default-class="btn-gradient-hr"
+        :is-loading="isLoading"
+        :disabled="isLoading"
+      >
+        Fund Campaign
+      </BaseButton>
```

Comment:
The new props is not really used by others.
Maybe you can create a new reusable button that is calling basebutton. Like this:

const defaultClasss= 'btn-gradient-hr'
const loadingClass = 'bg-disabled h-9 px-4 rounded-[6px] text-white'
<NewButton>
<BaseButton
        type="submit"
        :class="isLoading ? loadingClass : defaultClass"
        :disabled="isLoading"
      >
<NewButton>

Above is only applicable if you think there will be similar button in the future.
Else, revert the changes